app/controllers/application_controller.rb

```diff
@@ -44,6 +44,12 @@ def render_update(record, serializer_class)
     end
   end
 
+  def process_index(base, serializer_class, opts = {})
+    scope = paginate(policy_scope(base))
+    meta = { :total => scope.count, :per_page => scope.per_page, :page => scope.current_page }
+    render :json => serializer_class.new(scope, opts.merge(:meta => meta))
```

Comment:
```suggestion
    render :json => serializer_class.new(scope, {meta: meta}.deep_merge(opts))
```
1. reverse the order, so `opts` will take precedence
2. Use `deep_merge`, otherwise the whole `meta` tag would be replaced.